controllers/workouts-controller.js

```diff
@@ -11,7 +11,11 @@ function getAllWorkouts(request, response, next) {
 
 
 function getWorkoutByLevel(request, response, next) {
-    fetchWorkoutByLevel(request.params.level).then((workout) => {
+    const { level } = request.params;
+    fetchWorkoutByLevel(level).then((workout) => {
+        if (isNaN(level)) {
+            return response.status(400).send({ message: "Invalid query" });
+        }
```

Comment:
This if statement would be better done in workouts-model. In there, you can check if the workoutLevel variable is not a number (that is, `if(isNaN(workoutLevel)`), and if that's the case, return a rejected promise with the status and message you want to return. If the parameter given was invalid, there might be a chance that it didn't make it into this then block, but we're always going to go to the model, so it'd be better to test there.